### ISSUE TITLE
Post-migration event

### ISSUE THREAD

**Issue by lonix1** [dotnet/efcore#26976]
I didn't know whether to create a feature request or question.

## Ask a question
I asked [on SO](https://stackoverflow.com/q/70312705/9971404) first.

I'd like to run code after all migrations are applied, and for that to happen whether I migrate programmatically (`context.Database.Migrate()`) or by the CLI (`dotnet ef database update`).

I hoped there were before/after events like there are for [other parts of EF](https://docs.microsoft.com/en-us/ef/core/logging-events-diagnostics/events), but couldn't find any.

I could add a blank migration, name it "999999" to ensure it's run last, and put the code in `Up()`. But that would be run only once. I need to run code in an "after event" sort of scenario, every time.

Is there a formal way, or a workaround?

## Include provider and version information

EF Core version: 5.0.11
Database provider: sqlite and postgres
Target framework: dotnet5
Operating system: linux
IDE: vscode latest

**Comment by ajcvickers** [comment]
Duplicate of #24710.

@lonix1 Make sure to vote (👍) for that issue.

**Comment by lonix1** [comment]
Sorry @ajcvickers, didn't see that issue.

**Comment by ajcvickers** [comment]
@lonix1 No problem; it can be hard to find duplicates. :-)
